homeassistant/components/aladdin_connect/__init__.py

```diff
@@ -1,48 +1,53 @@
-"""The aladdin_connect component."""
+"""The Aladdin Connect Genie integration."""
 
-import logging
-from typing import Final
-
-from AIOAladdinConnect import AladdinConnectClient
-import AIOAladdinConnect.session_manager as Aladdin
-from aiohttp import ClientError
+from __future__ import annotations
 
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
-from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
-from homeassistant.helpers.aiohttp_client import async_get_clientsession
-
-from .const import CLIENT_ID, DOMAIN
+from homeassistant.helpers import aiohttp_client, config_entry_oauth2_flow
 
-_LOGGER: Final = logging.getLogger(__name__)
+from . import api
+from .const import CONFIG_FLOW_MINOR_VERSION, CONFIG_FLOW_VERSION
 
-PLATFORMS: list[Platform] = [Platform.COVER, Platform.SENSOR]
```

Comment:
This will "strand" the sensors for the existing users.... should the update remove them?

Reply:
Oh this is a major issue, this should be resolved as well